Week3Project/Week3Project/users.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Week3Project
+{
+    class User
+    {
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+        public int favoriteNumber { get; set; }
+        public int numberOfPets { get; set; }
+        public string namesOfPets { get; set; }
+
+        public string AnswerQuestion(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+
+        }
+        public int AnswerQuestionWithInteger(string question)
+        {
+            string stringAnswer = AnswerQuestion(question);
```

Comment:
Excellent re-use of a method that had already solved the problem of asking the user for input 👍